instrumentation/jetty/jetty-8.0/src/main/java/io/opentelemetry/instrumentation/auto/jetty/Jetty8HandlerInstrumentation.java

```diff
@@ -54,16 +56,19 @@ public JettyHandlerInstrumentation() {
   public String[] helperClassNames() {
     // order matters here because subclasses (e.g. JettyHttpServerTracer) need to be injected into
     // the class loader after their super classes (e.g. Servlet3HttpServerTracer)
+    String commonPackageName = "io.opentelemetry.instrumentation.auto.servlet.v3";
+    String servlet30PackageName = "io.opentelemetry.instrumentation.auto.servlet.v3_0";
     return new String[] {
       "io.opentelemetry.instrumentation.servlet.HttpServletRequestGetter",
       "io.opentelemetry.instrumentation.servlet.ServletHttpServerTracer",
-      "io.opentelemetry.instrumentation.auto.servlet.v3_0.Servlet3HttpServerTracer",
-      "io.opentelemetry.instrumentation.auto.servlet.v3_0.TagSettingAsyncListener",
-      "io.opentelemetry.instrumentation.auto.servlet.v3_0.CountingHttpServletRequest",
-      "io.opentelemetry.instrumentation.auto.servlet.v3_0.CountingHttpServletResponse",
-      "io.opentelemetry.instrumentation.auto.servlet.v3_0.CountingHttpServletResponse$CountingServletOutputStream",
-      "io.opentelemetry.instrumentation.auto.servlet.v3_0.CountingHttpServletResponse$CountingPrintWriter",
-      packageName + ".JettyHttpServerTracer",
+      commonPackageName + ".CountingHttpServletRequest",
```

Comment:
@trask suggested we keep the full names so IDE can refactor them properly. What do you think?

Reply:
Good point, I haven't thought about that.

Reply:
just fyi: https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/890#issuecomment-673754096